stetho-repl-js/README.md

```diff
@@ -0,0 +1,223 @@
+# Stetho's Javascript Console
+
+This [Stetho](https://facebook.github.io/stetho) plugin adds a javascript console by embedding Mozilla's [Rhino](https://github.com/mozilla/rhino).
+
+## Set-up
+
+### Download
+Download [the latest JARs](https://github.com/facebook/stetho/releases/latest) or grab via Gradle:
+```groovy
+compile 'com.facebook.stetho:stetho-repl-js:1.1.1'
+```
+or Maven:
+```xml
+<dependency>
+  <groupId>com.facebook.stetho</groupId>
+  <artifactId>stetho-repl-js</artifactId>
+  <version>1.1.1</version>
+</dependency>
+```
+
+Make sure that you depend on the main `stetho` dependency too.
+
+### Putting it together
+The javascript integration is similar to standard Stetho integration.
+The main difference is with the WebKitInspector used.
+There is a simple initialization step which occurs in your `Application` class:
+
+```java
+public class MyApplication extends Application {
+  public void onCreate() {
+    super.onCreate();
+    JavaScriptRuntimeBuilder jsRuntimeBuilder = new JavaScriptRuntimeBuilder(this);
```

Comment:
Also, we should do this construction lazily inside the inspector modules provider.  The intention of this awkward interface is to make it so that Stetho.initialize does no real work at all until a client actually connects.